src/main/java/frc/robot/commands/ToggleFieldAbsoluteCommand.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.DriveSubsystem;
+
+/** An example command that uses an example subsystem. */
+public class ToggleFieldAbsoluteCommand extends EntechCommandBase {
+  @SuppressWarnings({ "PMD.UnusedPrivateField", "PMD.SingularField" })
+  private final DriveSubsystem drive;
+  private boolean isFinished = false;
+
+  /**
+   * Creates a new ToggleFieldAbsoluteCommand.
+   *
+   * @param Drive The subsystem used by this command.
+   */
+  public ToggleFieldAbsoluteCommand(DriveSubsystem Drive) {
+    super(Drive);
+    drive = Drive;
```

Comment:
when you rename the input to "drive", you'll have to change this to either (this.drive = drive) to avoid the shadowed variable